R/get_draws_df.R

```diff
@@ -113,17 +131,17 @@ get_draws_df <- function(ww_data,
       flag_as_ww_outlier = NA,
       exclude = NA
     ) |>
-    dplyr::select(-t)
+    dplyr::select(-"t")
 
   site_level_rt_draws <- draws |>
-    tidybayes::spread_draws(r_site_t[site_index, t]) |>
-    dplyr::rename(pred_value = r_site_t) |>
+    tidybayes::spread_draws(!str2lang("r_site_t[site_index, t]")) |>
```

Comment:
Just a missing exclamation point @gvegayon 
```suggestion
    tidybayes::spread_draws(!!str2lang("r_site_t[site_index, t]")) |>
```